### PR TITLE
Remove deprecated batchpertrace package/module

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -265,6 +265,10 @@ updates:
     directory: "/internal/coreinternal"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/internal/docker"
+    schedule:
+      interval: "weekly"
```
Comment (codeboten) [inline .github/dependabot.yml:271]: Is this related to the same change?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -497,10 +501,6 @@ updates:
     directory: "/receiver/sapmreceiver"
     schedule:
       interval: "weekly"
-  - package-ecosystem: "gomod"
-    directory: "/receiver/scrapererror"
-    schedule:
-      interval: "weekly"
```
Comment (codeboten) [inline .github/dependabot.yml:503]: Was this also deprecated?

**Review by codeboten — COMMENTED** [review]
Couple of questions in the comments, please add a note to the changelog.

File: .github/dependabot.yml (hunk 1 above)
Comment (bogdandrutu) [reply .github/dependabot.yml:271]: It was removed, I just run `make gendependabot`

File: .github/dependabot.yml (hunk 2 above)
Comment (bogdandrutu) [reply .github/dependabot.yml:503]: It was removed, I just run `make gendependabot`

**Review by mx-psi — APPROVED** [review]
Code looks good, we should add a release note
